lib/core/services/identity-user.service.interface.ts

```diff
@@ -16,11 +16,14 @@
  */
 
 import { Pagination } from '@alfresco/js-api';
+import { InjectionToken } from '@angular/core';
 import { Observable } from 'rxjs';
 import { IdentityGroupModel } from '../models/identity-group.model';
 import { IdentityRoleModel } from '../models/identity-role.model';
 import { IdentityUserModel } from '../models/identity-user.model';
 
+export const IDENTITY_USER_TOKEN = new InjectionToken<IdentityUserServiceInterface>('identity-user-token');
```

Comment:
can you put this in a new file? identity-user-service.token.ts for example? Also the `identity-user-token` should be `identity-user-service-token` ?

Reply:
renamed and moved